tests/functional/test_accounts.py

```diff
@@ -364,6 +364,16 @@ def test_account_comparison_to_non_account(core_account):
     assert core_account != "foo"
 
 
+def test_create_account(test_accounts):
+    assert len(test_accounts) == DEFAULT_NUMBER_OF_TEST_ACCOUNTS
+
+    created_acc = test_accounts.containers["test"].create_account()
+
+    assert len(test_accounts) == DEFAULT_NUMBER_OF_TEST_ACCOUNTS + 1
```

Comment:
Do we need to add this to the account list?
I am thinking it should work more like how impersonated accounts work when using `ape-hardhat` or `ape-foundry`.

Reply:
Yeah, this is my opinion too. I don't think it should be added to the list, instead there are "genesis accounts" in `TestAccountContainer` and impersonated accounts that come from `TestAccountContainer.create_account()` or `TestAccountContainer.__getitem__(addr)`